common/changes/@microsoft/rush/fix-cobuild-build-cache_2024-03-05-17-28.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "fixes an issue where cobuilds would write success with warnings as success cache entries unexpectedly.",
+      "type": "patch"
```

```suggestion
      "comment": "Fixes an issue where cobuilds would write success with warnings as successful cache entries.",
      "type": "none"
```

The Rush release type is dictated by the version-policies.json file

---

Thanks 😅 ! New to the versioning side of things.